SMARTAGRIFOOD/wineTraceability/vineyardBlock/NOTES.YAML

```diff
@@ -0,0 +1,2 @@
+notesHeader:
+  Vineyard block data model definition is currently in incubated stage (work in progress). It is incomplete and large changes may occur at this stage, so it is highly advisable to not use the model until the version 1.0 is reached.
```

Comment:
We start the versions in 0.X, so evolve the model here in the incubated and go for official publication once you have something working. Extension of the model is easy. It is important not to create backward incompatible versions (initially we don't do them)